sherpa/astro/data.py

```diff
@@ -403,6 +403,19 @@ def __setstate__(self, state):
             self.header = None
         self.__dict__.update(state)
 
+    def _validate(self, name, energy_lo, energy_hi, ethresh):
+        """
+        Default validation: do nothing and just pass the arrays through.
```

Comment:
I don't think this comment is right. It looks to me like the default is to actually do the validation. Or am I reading this wrong?